packages/perspective-viewer-d3fc/src/js/legend/styling/enableDragging.js

```diff
@@ -0,0 +1,65 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2017, the Perspective Authors.
+ *
+ * This file is part of the Perspective library, distributed under the terms of
+ * the Apache License 2.0.  The full license can be found in the LICENSE file.
+ *
+ */
+
+import * as d3 from "d3";
+import {isElementOverflowing} from "../../utils/utils";
+import {getChartElement} from "../../plugin/root";
+
+export function enableDragging(element) {
+    const node = element.node();
+    node.style.cursor = "move";
+
+    const drag = d3.drag().on("drag", function() {
+        overrideDefaultResizingBehaviour(node);
```

Comment:
`overrideDefaultResizingBehaviour` adds an event handler, so it probably doesn't make sense to call it repeatedly from inside this event handler.

Reply:
Fixed.